apps/trading/components/navbar/navbar.tsx

```diff
@@ -88,7 +84,7 @@ export const Navbar = ({ theme = 'system' }: { theme?: Theme }) => {
         </div>
         <NavbarMobileButton
           onClick={() => {
-            if (isConnected) {
+            if (status === 'connected') {
```

Comment:
(non-blocking): it'd be great to have here some sort of status type to prevent any typos

Reply:
I'm not 100% convinced we should always use enums for this sort of thing. I quite like not to have to import an enum for a simple type comparison. The string will autocomplete because status is correctly typed. I think @dexturr feels strongly about using enums and I'm happy to make the change